lib/solidus_segment/serializers/address_serializer.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+module SolidusSegment
+  module Serializers
+    class AddressSerializer < Hash
```

Comment:
Extending a core class is very risky and error prone, why wouldn't we use a decorator and wrap a Hash?

Reply:
@filippoliverani 👍 I've changed it.